paddle/operators/conv_op.cc

```diff
@@ -70,6 +70,13 @@ void ConvOp::InferShape(framework::InferShapeContext* ctx) const {
 framework::OpKernelType ConvOp::GetExpectedKernelType(
     const framework::ExecutionContext& ctx) const {
   bool use_cudnn = ctx.Attr<bool>("use_cudnn");
+  use_cudnn &= platform::is_gpu_place(ctx.GetPlace());
```

Comment:
I think `use_cudnn` means that user wants to force use cudnn operator, if env does not support cudnn, the framework should throw an error but not implicit change to use other operator.

Reply:
Agree with the comment.  
We need a default policy right now, by default, we need to set `use_cudnn` True.
As we discussed offline, we reach the agreement that place is also a user configuration. we need to **overwrite our default policy**, not user specified.

Reply:
ok, I get the point, seems we still need a priority mechanism to resolve such kind of problem in the future.